Add string versions of compress/uncompress.

treeform opened the pull request:
I need to add these two functions when I use this excellent library. I wish they just existed here.

Araq said that cast from seq[byte] to string should always work. This does a zero copy operation and just works.

Thanks!

jangko commented:
when status-im/nim-faststreams#6 merged, it should fixes issue #2 also

stefantalpalaru commented on snappy.nim:
```diff
@@ -68,6 +68,7 @@ func load64(b: openArray[byte]): uint64 {.inline.} =
 func load64(b: openArray[byte], i: int): uint64 =
   result = load64(b[i..<i+8])
 
+import typetraits
```
Leftover from debugging?

jangko replied:
yup, i will remove it

treeform committed:
Add string versions of compress/uncompress.

I need to add these two functions when I use this excellent library. I wish they just existed here.

Araq said that cast from seq[byte] to string should always work. This does a zero copy operation and just works.

Thanks!
